tests/resources/charts/gateway-test/values.yaml

```diff
@@ -0,0 +1,19 @@
+global:
+  containerRegistry:
+    path: "europe-docker.pkg.dev/kyma-project"
```

Comment:
Please update repo name and image to use local k3d repository as we do for compass-runtime-agent tests

Reply:
as we agreed offline, this issue will be addressed in separate PR